felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/redis/RedisKey.kt

```diff
@@ -18,6 +19,11 @@ sealed class RedisKey(open val uuid: String) {
         fun of(uuid: String, eventname: EventName): RedisKey {
             return TransactionKey(uuid, eventname)
         }
+
+        // @TODO ikke bra nok
```

Comment:
hva er ikke bra nok?

Reply:
Jeg er ikke fornøyed med Redis.of() syntaksen. Er ikke klar nok hva slags nøkel blir det. Men det tar jeg i en annen refaktorerigssesjon sennere.